cg/apps/lims/orderform.py

```diff
@@ -83,35 +86,20 @@ def parse_orderform(excel_path: str) -> dict:
     return data
 
 
-def get_document_title(workbook: xlrd.book.Book, orderform_sheet: xlrd.sheet.Sheet) -> str:
-    """Get the document title for the order form."""
-    if "information" in map(str.lower, workbook.sheet_names()):
-        information_sheet = get_information_sheet(workbook)
-        document_title = information_sheet.row(0)[2].value
-        return document_title
-
-    document_title = orderform_sheet.row(0)[1].value
-    return document_title
-
+def get_document_title(workbook: Workbook, orderform_sheet: Worksheet) -> str:
+    """Get the document title for the order form.
 
-def get_information_sheet(workbook: xlrd.book.Book) -> xlrd.sheet.Sheet:
-    """Get the excel sheet named Information."""
+    Openpyxl use 1 based counting
+    """
+    for sheet_number, sheet_name in enumerate(workbook.sheetnames):
+        if sheet_name.lower() == "information":
+            information_sheet: Worksheet = workbook[sheet_name]
+            document_title = information_sheet.cell(1, 3).value
+            return document_title
 
-    sheet_name = None
-    sheet_names = workbook.sheet_names()
-    for name in ["Information", "information"]:
-        if name in sheet_names:
-            sheet_name = name
-            break
-    if sheet_name is None:
-        raise OrderFormError("Information sheet not found in Excel file")
-
-    sheet = workbook.sheet_by_name(sheet_name)
-
-    if not sheet:
-        raise OrderFormError(f"'information' sheet not found in: {workbook.sheet_names()}")
-
-    return sheet
+    # By looking at the test file this will not work
```

Comment:
if this finished and working? If so, this is not a necessary comment

Reply:
thx